processor/batchprocessor/splitmetrics.go

```diff
@@ -133,8 +133,11 @@ func splitMetric(ms, dest pdata.Metric, size int) (int, bool) {
 	case pdata.MetricDataTypeGauge:
 		return splitNumberDataPoints(ms.Gauge().DataPoints(), dest.Gauge().DataPoints(), size)
 	case pdata.MetricDataTypeSum:
+		dest.Sum().SetAggregationTemporality(ms.Sum().AggregationTemporality())
+		dest.Sum().SetIsMonotonic(ms.Sum().IsMonotonic())
 		return splitNumberDataPoints(ms.Sum().DataPoints(), dest.Sum().DataPoints(), size)
 	case pdata.MetricDataTypeHistogram:
+		dest.Histogram().SetAggregationTemporality(ms.Histogram().AggregationTemporality())
```

Comment:
It would help to add a testcase for this, to avoid further regressions

Reply:
+1

Reply:
@dashpole please do a followup PR.